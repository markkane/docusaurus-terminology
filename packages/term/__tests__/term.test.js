import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Term from '../src';

test('Term renders', () => {
  function Wrapper() {
    return (
      <MemoryRouter>
        <Term reference="test" popup="test">test</Term>,
      </MemoryRouter>
    )
  };
  const component = renderer.create(
    <Wrapper />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
