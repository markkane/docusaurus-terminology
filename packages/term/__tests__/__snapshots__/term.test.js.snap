// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Term renders 1`] = `
Array [
  <a
    aria-describedby={null}
    className=""
    href="test"
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onMouseLeave={[Function]}
    onMouseOver={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    title={null}
  >
    <span
      style={
        Object {
          "fontWeight": "bold",
        }
      }
    >
      test
    </span>
  </a>,
  ",",
]
`;
